### ISSUE TITLE
Doesn't even begin to work in any shape or form, unfortunately.

### ISSUE THREAD

**Issue by YShane** [MicrosoftIsDumb/Defender-for-Cloud-Apps-Proxy-Bypass#1]
Doesn't work with any of the user agent strings. Or any others, for that matter.

**Comment by joaociocca** [comment]
yup, I'd say that if it worked one day, MSRC may have told the repo owner that it wasn't worth fixing, but fixed it...

**Comment by MicrosoftIsDumb** [comment]
Just tested again and seems to still be a functional bypass. The only way I know of to prevent this is to explicitly block user agent strings in the defender for cloud platform (Which is a rabbit hole). This may have have unforeseen consequences with official MS client apps.

This (these user agent strings) are the reason that you are allowed to copy and paste when using the teams application.
The official teams application is simply an electron client app with the teams user agent.

That being said I'm not an expert in Microsoft Defender so I suppose there could be a few ways within that tool to mitigate this somehow without impacting the teams client for example.

I would recommend verifying your user agent. You may search "What's my user agent" on google and it will tell you.

Also, I've noticed that you cannot access teams in the browser while using the teams user agent. But, you can access it fine when using a different user agent. 

If there's any specifics you can provide I can try to get to the bottom of why this isn't working for you.

**Comment by joaociocca** [comment]
I guess this could be a misinterpretation of descriptions, then. When you say "this bypasses Cloud for Apps proxy", I understand that it bypasses proxy blocks put in place using Cloud for Apps.
The way you explained now, I understand that it's meant as a way to bypass copying/pasting things into/from specific MS sites/apps, like Teams.

**Comment by SynapseScribe** [comment]
ain't working with any user agent. the moment i switch to https://outlook.office.com/mail/ it stops logging in

**Comment by pagobman-netizen** [comment]
Doesn't work anymore since the owner "detect bypass session control" (early July 2025)
even though i changed any string-code on user agent switcher, made me frustrated when i doin my work at home

![Image](https://github.com/user-attachments/assets/475caea2-d8ed-4c88-9a67-99eec40b7a42)
